pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -66,7 +81,7 @@ func newBundleUnpackResult(lookup *operatorsv1alpha1.BundleLookup) *BundleUnpack
 	}
 }
 
-func (c *ConfigMapUnpacker) job(cmRef *corev1.ObjectReference, bundlePath string, secrets []corev1.LocalObjectReference) *batchv1.Job {
+func (c *ConfigMapUnpacker) job(cmRef *corev1.ObjectReference, bundlePath string, secrets []corev1.LocalObjectReference, ipAnnotations map[string]string) *batchv1.Job {
```

Comment:
I think this should have a `timeout` parameter instead. It's unnecessary to depend on the entire annotations map.

Reply:
Yep, this can just be a timeout. I was trying to keep all the bundle-unpack-timeout annotation logic grouped in one place but I can parse and pass only what we need right here.